chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md

```diff
@@ -750,7 +750,7 @@ It is reasonable to ask at this point, "Why can I not just write down matrix ver
 * Matrix calculus allows us to write the derivatives of matrix expressions in concise ways.
 
 ## Exercises
-1. Given a row vector $\boldsymbol{\beta}$, compute the derivatives of both $f(\mathbf{x}) = \boldsymbol{\beta}\mathbf{x}$ and $g(\mathbf{x}) = \mathbf{x}^\top\boldsymbol{\beta}^\top$.  Why do you get the same answer?
+1. Given a row vector $\boldsymbol{\beta}^\top$, compute the derivatives of both $f(\mathbf{x}) = \boldsymbol{\beta}^\top\mathbf{x}$ and $g(\mathbf{x}) = \mathbf{x}^\top\boldsymbol{\beta}$.  Why do you get the same answer?
```

Comment:
Can you change it to column vector as well?

Reply:
The transpose of x in g(x) is a row vector, so it need the column one, which is the beta without transpose, to do dot product. 
Do I get your point?

Reply:
The transpose of x in g(x) is a row vector which need a column one, the beta without transpose, to do dot product, so I get rid of the transpose of beta in g(x).
Do I get the point?

Reply:
Both x and beta are column vectors. So yes your modification is correct, while can you change the "Given a row vector ..." to "Given a column vector..."?

Reply:
Ok!